rero_ils/modules/circ_policies/api.py

```diff
@@ -373,14 +373,17 @@ def get_reminder(self, reminder_type=DUE_SOON_REMINDER_TYPE, idx=0):
             return reminders[idx]
 
     @classmethod
-    def can_request(cls, item, **kwargs):
-        """Check if the cipo corresponding to item/patron allow request.
+    def can_request(cls, record, **kwargs):
+        """Check if the cipo corresponding to record/patron allow request.
 
-        :param item : the item to check
+        :param record : the record to check (Item or Holding)
         :param kwargs : To be relevant, additional arguments should contains
                         'patron' argument.
         :return a tuple with True|False and reasons to disallow if False.
         """
+        from ..holdings.api import Holding
```

Comment:
Check if you can move these `imports` to the top of the file.

Reply:
Unfortunately I cannot. I have already tried and I get errors (unrelated to this module) when running the tests.